src/driver/driver.rs

```diff
@@ -17,4 +17,14 @@ extern crate rustdoc as this;
 #[cfg(rustc)]
 extern crate rustc_driver as this;
 
+#[cfg_attr(target_os = "nacl", main_link_name = "nacl_main")]
 fn main() { this::main() }
+
+#[cfg(target_os = "nacl")]
+#[link(name = "ppapi_cpp", kind = "static")]
+#[link(name = "ppapi_simple_cpp", kind = "static")]
+#[link(name = "ppapi_stub", kind = "static")]
+#[link(name = "cli_main", kind = "static")]
+#[link(name = "tar", kind = "static")]
+#[link(name = "nacl_spawn", kind = "static")]
```

Comment:
Actually, `rust-ppapi` (or any pure Rust wrapper over PPAPI) can't be used at all with `ppapi_cpp`; both define the main PPAPI functions: `PPP_InitializeModule`, `PPP_ShutdownModule`, and `PPP_GetInterface`.